<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>429</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="panelwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>700</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.216, y1:0.232727, x2:1, y2:1, stop:0 rgba(255, 219, 88, 255), stop:1 rgba(255, 255, 255, 255));}</string>
    </property>
    <widget class="QWidget" name="bgwidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>700</width>
       <height>471</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.216, y1:0.232727, x2:0.847, y2:0.847, stop:0 rgba(255, 121, 63, 255), stop:1 rgba(255, 218, 121, 255));}</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>251</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Sans Pro</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/Eggsify-logo-01.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>D:/Pictures/Graphic Design/Eggsify/Eggsify-logo-small-01.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sizeWeightBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color:rgb(249, 249, 249)
</string>
      </property>
      <property name="text">
       <string>Size and Weight Classifcation </string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="qualityBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color:rgb(249, 249, 249)
</string>
      </property>
      <property name="text">
       <string>Quality Classifcation </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="D:/Pictures/Graphic Design/Eggsify/eggsify-logo.qrc"/>
 </resources>
 <connections/>
</ui>
